corehq/apps/cloudcare/static/cloudcare/js/util.js

```diff
@@ -103,6 +103,13 @@ var showSuccess = function (message, location, autoHideTime) {
     _show(message, location, autoHideTime, "alert alert-success");
 };
 
+var showRenderedForm = function (message, location, autoHideTime) {
```

Comment:
`autoHideTime` doesn't seem to be used. also some of the parameters i think are misleading. message is i believe the xml of the form. also location is confusing since location is already overloaded in HQ. i'd lean towards using something like `$container` to signify it's a jquery element. that would simplify it to something like this:

```
function (xml, $container) {
  $container.html(xml || "Could not render form");
}
```